dlk/python/dlk/templates/src/func/conv2d.cpp

```diff
@@ -45,9 +45,6 @@ void conv3x3_kn2row(T input[],
   assert(kh == 3 && kw == 3);
   assert(MAX_SIZE_KN2ROW_BUFFER_PER_LAYER >= oc * kh * kw * ih * iw);
 
-  // need to initialize output
```

Comment:
Is it possible to remove this `std::memset`? According to the code, it looks impossible for me.

Reply:
As a result of my detailed examination, you were right. Very sorry. I fixed it.